pkg/kubernetes/scripts/policy.js

```diff
@@ -190,7 +190,69 @@
 
                 return methods.create([binding], namespace);
             }
+            function removeRoleFromPolicyBinding(policyBinding, project, role, subject) {
```

Comment:
This is actually about 'removeSubjectToPolicyBinding' and should be named that way. The role may or may not exist already.